chain/txValidator.go

```diff
@@ -387,10 +387,17 @@ func VerifyTransactionWithLedger(txn *transaction.Transaction, height uint32) er
 		if err != nil {
 			return err
 		}
+		if len(registrant) == 0 {
+			return fmt.Errorf("can not transfer unregistered name")
+		}
 		if bytes.Equal(registrant, pld.Recipient) {
 			return fmt.Errorf("can not transfer names to its owner")
 		}
-		if !bytes.Equal(registrant, pld.Registrant) {
```

Comment:
Are we missing `pld.Registrant` verification here?

Reply:
fixed